spec.bs

```diff
@@ -1260,8 +1248,35 @@ an [=ordered map=] whose [=map/keys=] are [=strings=] and whose [=map/values=] a
     1. Set |global|'s
       [=InterestGroupBiddingScriptRunnerGlobalScope/is component auction=] to |isComponentAuction|.
     1. Set |global|'s [=InterestGroupBiddingScriptRunnerGlobalScope/interest group=] to |ig|.
-    1. Return the result of [=evaluating a script=] with |global|, |script|, "`generateBid`",
+    1. Let |startTime| be the current time.

```

Comment:
Should "current time" be current wall time https://dontcallmedom.github.io/webdex/c.html#current%20wall%20time%40%40hr-time%25%25dfn, as recommended elsewhere in other PRs and the spec feedback doc?

Reply:
Yes, that seems fine.  Looks like recommended coarsening is 100us which is suitably below our 1ms precision.  I changed the remaining two references to current wall time.

Reply:
That 1ms precision is one of the things that isn't actually specified until |browserSignals| becomes a WebIDL dictionary. Right now it's as plausible to keep the fractional milliseconds as to truncate them. Fixing this belongs in another PR, of course.